public/app/features/logs/components/LogRowContext.tsx

```diff
@@ -261,6 +301,12 @@ export const LogRowContext: React.FunctionComponent<LogRowContextProps> = ({
             logsSortOrder={logsSortOrder}
           />
         )}
+        <div className={cx(title, sizes)}>
+          <h5>Show Context</h5>
```

Comment:
```suggestion
          <h5>Log context</h5>
```
Maybe Log context?

Reply:
Good Idea! Renamed in https://github.com/grafana/grafana/pull/56283/commits/519a1ecbc56ffe5a90d50eadbee334b44a5b3e2b